### PR TITLE
Feat/ add SI related props and events

### PR DESCRIPTION
**Pull request by othellashao**
# COMPLETES #
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-443833
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-443639
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-443617
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-443614

## This pull request addresses

Add basic props and events for SI feature.

## by making the following changes

1. update `siLanguages` from controls of lcous dto
2. update self interpreter information from self's controls of locus dto
3. add apis: `querySupportLanguages`, `getInterpreters`, `updateInterpreters`, `changeDirection`
4. auto query interpretation languages if self become host

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [x] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by mickelr — APPROVED** [review]
LGTM

File: packages/@webex/plugin-meetings/src/interpretation/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+### Interpreter functionality
+
+The following are methods available to the interpreters of a meeting.
+
+```javascript
+//Change direction of interpretation for an interpreter participant
+interpretation.changeDirection();
+
+//Handover between interpreters, will implement them later
+interpretation.handover(participantId)
```
[the remark this answers is not in the document]
Comment (othellashao) [reply packages/@webex/plugin-meetings/src/interpretation/README.md:49]: @mickelr OK, let me change it

File: packages/@webex/plugin-meetings/src/interpretation/index.ts [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        this.querySupportLanguages();
+      }
+    });
+  },
+
+  /**
+   * Calls this to clean up listeners
+   * @returns {void}
+   */
+  cleanUp() {},
```
Comment (Coread) [inline packages/@webex/plugin-meetings/src/interpretation/index.ts:61]: question: this cleanup function is empty. Should it be empty?

File: packages/@webex/plugin-meetings/src/constants.ts [hunk 3]
```diff
@@ -592,13 +594,22 @@ export const BREAKOUTS = {
   DEFAULT_TTL: 30,
 };
 
+export const INTERPRETATION = {
+  EVENTS: {
+    SUPPORT_LANGUAGES_UPDATE: 'SUPPORT_LANGUAGES_UPDATE',
+  },
+  INTERPRETER_CONTROL_SI_SUPPORTED: 'INTERPRETER_CONTROL_SI_SUPPORTED',
+  HOST_CONTROL_SI_SUPPORTED: 'HOST_CONTROL_SI_SUPPORTED',
```
Comment (sundayu) [inline packages/@webex/plugin-meetings/src/constants.ts:602]: These are no longer needed.  please remove.
The `deviceCapabilities` will be used directly in the client.

Comment (othellashao) [reply packages/@webex/plugin-meetings/src/constants.ts:602]: @sundayu OK

File: packages/@webex/plugin-meetings/src/interpretation/index.ts (hunk 2 above)
Comment (othellashao) [reply packages/@webex/plugin-meetings/src/interpretation/index.ts:61]: @Coread thank you for figure it out. I'd add the stopListening in cleanUp

**Review by Coread — APPROVED** [review]

**Pull request merged**